Tasks/Common/docker-common-v2/containerconnection.ts

```diff
@@ -23,7 +23,13 @@ export default class ContainerConnection {
     private oldDockerConfigContent: string;
 
     constructor() {
-        this.dockerPath = tl.which("docker", true);
+        if(process.env["RUNNING_ON"] == "KUBERNETES") {
```

Comment:
You need to increment other tasks version which are depending on  docker-common-v2